1.9/administration/installing/cloud/gce.md

```diff
@@ -267,7 +267,7 @@ Do not change this parameter. It is required by the Google Cloud SDK.
 Specify the filename for the DC/OS installer. Default `dcos_generate_config.sh`.
 
 ### dcos_installer_download_path
-Specify the location of where the DC/OS installer is available from [dcos.io](https://dcos.io). Default: `https://downloads.dcos.io/dcos/testing/master/{{ dcos_installer_filename }}?_ga=1.3100023.674486911.1475259096`. The value of `{{ dcos_installer_file }}` is described above.
+Specify the location of where the DC/OS installer is available from [dcos.io](https://dcos.io). Default: `https://downloads.dcos.io/dcos/stable/{{ dcos_installer_filename }}`. The value of `{{ dcos_installer_file }}` is described above.
```

Comment:
I think you're confusing the `_ga` param with the git sha... 
The above change says `https://downloads.dcos.io/dcos/stable/commit/0ce03387884523f02624d3fb56c7fbe2e06e181b/{{ dcos_installer_filename }}` which includes the git sha, but you don't have it here.

Neither should have the `_ga` tag. That's just a flag for google analytics that get's added at runtime by the google javascript for link tracking.